lib/site_prism/waiter.rb

```diff
@@ -18,5 +18,11 @@ def self.default_wait_time
       warn 'default_wait_time is now deprecated. This will be removed in an upcoming release.'
       Capybara.default_max_wait_time
     end
+
+    def self.wait_with_capybara(timeout: nil, or_raise: nil)
+      timeout = Capybara.default_max_wait_time if timeout.nil?
```

Comment:
`timeout = timeout || Capybara.default_max_wait_time`

Reply:
```ruby
timeout ||= Capybara.default_max_wait_time
```
Maybe this?